### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by grunweg** [comment]
!bench

**Review by grunweg — COMMENTED** [review]
Thanks for the clean-up! I have three minor comments/questions, but no blocking objections. Let's have another pair of eyes.
maintainer merge

File: Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean
Comment (Vierkantor) [inline Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean:107]: Should these renames come with a `@[deprecated] alias`?

Comment (grunweg) [reply Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean:107]: My understanding is that explicit instance names do not get deprecation aliases. (That said: if you feel there should be consensus on it, feel free to further the zulip thread on a deprecation policy. I intend to return it it, but not in before three weeks.)

File: Mathlib/Order/Filter/Basic.lean
Comment (Vierkantor) [reply Mathlib/Order/Filter/Basic.lean:1546]: They look good to me. I don't see any likely issues with looping assumptions or problematic matching. We've had them for almost a year so I don't think it's dangerous.

Comment (Vierkantor) [reply Mathlib/Order/Filter/Basic.lean:1546]: Note that `instance : Trans` is the Lean 4 equivalent of the `@[trans]` attribute.

**Review by Vierkantor — APPROVED** [review]
This looks good to me with @grunweg's comments. Thanks!

bors d+

File: Mathlib/RingTheory/WittVector/FrobeniusFractionField.lean
Comment (Ruben-VandeVelde) [reply Mathlib/RingTheory/WittVector/FrobeniusFractionField.lean:152]: Yeah, I don't think it's desirable to change this back

**Pull request closed without merging**
